docker-compose.yml

```diff
@@ -1,5 +1,25 @@
 version: "3.9"
 services:
+  elasticsearch7:
+    image: elasticsearch:7.16.2
+    ports:
+      - "9200:9200"
+    environment:
+      - discovery.type=single-node
+      - bootstrap.memory_lock=true
+      - "ES_JAVA_OPTS=-Xms512m -Xmx512m"
```

Comment:
It looks like for docker-compose, the `ES_JAVA_OPTS` was applied to both v7 and v8, but for GHA it was only used for one of the versions. Should it be used for both?

Reply:
I'm gonna take it out of GHA since it's probably not actually needed there. We do need it on both in docker compose though, since both 7 and 8 would fail with the out of memory error.